packages/create-pantheon-decoupled-kit/src/utils/taggedTemplateHelpers.ts

```diff
@@ -0,0 +1,39 @@
+export const taggedTemplateHelpers = {
+	/**
+	 * @param condition - a single key of an {@link Input}
+	 * @param value - the value to render if the condition is true
+	 * @returns the value if it a string, or an empty string
+	 */
+	if: (condition: unknown, value: string) => (condition ? value : ''),
```

Comment:
This one is new and I expect we might have more helpers similar to this one to prevent things like 

```ts
${data.tailwindcss ? 'x' : ''}
``` 
It becomes 
```ts
${utils.if(data.tailwindcss, 'x')}
```
Open to different names.